src/lib.rs

```diff
@@ -6,11 +6,11 @@ extern crate vorbis_sys;
 
 #[repr(C)]
 pub struct ov_callbacks {
-    pub read_func: extern fn(*mut libc::c_void, libc::size_t, libc::size_t, *mut libc::c_void)
-        -> libc::size_t,
-    pub seek_func: extern fn(*mut libc::c_void, ogg::ogg_int64_t, libc::c_int) -> libc::c_int,
-    pub close_func: extern fn(*mut libc::c_void) -> libc::c_int,
-    pub tell_func: extern fn(*mut libc::c_void) -> libc::c_long,
+    pub read_func: Option<extern fn(*mut libc::c_void, libc::size_t, libc::size_t, *mut libc::c_void)
```

Comment:
I'm not sure if the `read_func` callback might be `NULL`, at least it's not mentioned [here](https://xiph.org/vorbis/doc/vorbisfile/ov_callbacks.html

Reply:
I originally made this pull request to make the struct zero-able in vorbis-sys, but regardless of that, I don't see a reason to not use Option here, since Option<fn()> is the type that is actually layout/FFI-compatible with a C function pointer